frontend/public/components/cluster-settings/cluster-operator.tsx

```diff
@@ -90,15 +92,27 @@ const filters = [{
   })),
 }];
 
+export const ClusterOperatorStartGuide: React.SFC<ClusterOperatorStartGuideProps> = props =>
+  <div className="co-well">
+    <Button className="co-well__close-button" aria-label="Close" bsStyle="link" onClick={props.dismiss}><i className="pficon pficon-close" aria-hidden="true" /></Button>
```

Comment:
The button should be part of `StartGuide` so it is consistent for every guide you can dismiss.

Nice job making the button accessible :)